src/setupTools.js

```diff
@@ -144,6 +144,8 @@ export default function setupTools(store) {
   const availableTools = [
     { name: 'Pan', mouseButtonMasks: [1, 4] },
     { name: 'Zoom', mouseButtonMasks: [1, 2] },
+    { name: 'ZoomMouseWheel' },
+    { name: 'PanZoomMouseWheel', mouseButtonMasks: [1, 4] },
```

Comment:
Not sure you need the `mouseButtonMasks: [1, 4]`, have you tried without that?

Reply:
You're right, that's no needed here. Updated!